lib/parser/message.cc

```diff
@@ -35,6 +35,19 @@ std::ostream &operator<<(std::ostream &o, const MessageFixedText &t) {
 
 MessageFormattedText::MessageFormattedText(MessageFixedText text, ...)
   : isFatal_{text.isFatal()} {
+  va_list ap;
+  va_start(ap, text);
+  SetMessageFormattedText(text, ap);
+  va_end(ap);
+}
+
+MessageFormattedText::MessageFormattedText(MessageFixedText text, va_list ap)
+  : isFatal_{text.isFatal()} {
+  SetMessageFormattedText(text, ap);
+}
+
+void MessageFormattedText::SetMessageFormattedText(MessageFixedText text,
```

Comment:
I get where this member function name came from but it might be more clear if it were shorter -- we don't need to use the name of the class redundantly in the name of the member function if it's not a constructor or destructor.  Even "vsnprintf" might be better.

Reply:
OK

Reply:
Removed this